weldx/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-0.1.0.yaml

```diff
@@ -327,13 +329,10 @@ examples:
                 300.00000000000006]
               datatype: float64
               shape: [6]
-            time: !<asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-0.1.0>
-              start: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M0S
-              end: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M10S
-              freq: 2S
-              min: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M0S
-              max: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M10S
-            units: !<asdf://weldx.bam.de/weldx/tags/unit/unit-0.1.0> A
+            time: !<asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-0.1.0> {start: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M0S,
+              end: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M10S, freq: 2S, min: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M0S,
+              max: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-0.1.0> P0DT0H0M10S}
+            units: !<asdf://weldx.bam.de/weldx/tags/unit/unit-0.1.0> ampere
```

Comment:
I don't know why the formatting changed here but it doesn't really matter

Reply:
we can probably force it with `flowStyle: block`, I'll add that back in some other time